scripting/get5.sp

```diff
@@ -235,6 +237,8 @@ public void OnPluginStart() {
   LoadTranslations("common.phrases");
 
   /** ConVars **/
+  g_AllowTechPause =
+      CreateConVar("get5_allow_technical_pause", "1", "Whether or not Technical pauses are allowed");
```

Comment:
ultra-nitpick: having "Technical" randomly capitalized annoys me :) (same for the translation files)